src/ot-rcp/app.c

```diff
@@ -21,6 +21,7 @@
 #include <openthread/ncp.h>
 #include <openthread/diag.h>
 #include <openthread/tasklet.h>
+#include <openthread/logging.h>
```

Comment:
It's not quite clear to me why logging is needed here, as there seem to be no logging calls below :thinking: 

But it's detail, if there is a reason, fine by me.

Reply:
Ah. It's because I used `otPlatAssertFail` in multi-PAN, the pure OpenThread firmware doesn't implement a replacement for this function.